src/duckling/game/drawable/tile-block-drawable-drawer.ts

```diff
@@ -15,30 +15,28 @@ export function drawTileBlockDrawable(tileBlockDrawable : TileBlockDrawable, ass
         return null;
     }
     
-    let container = new Container();
-    container.addChild(_constructTileBlockSprite(tileBlockDrawable, baseTexture, assetService));
-    return container;
+    // let container = new Container();
+    return _constructTileBlockSprite(tileBlockDrawable, baseTexture, assetService);
+    // return container;
 }
 
 function _constructTileBlockSprite(tileBlockDrawable : TileBlockDrawable, baseTexture : BaseTexture, assetService : AssetService) : DisplayObject {
     let tileSize = getTileWidth(tileBlockDrawable, assetService);
-    let tiles = { x: tileBlockDrawable.size.x / tileSize, y: tileBlockDrawable.size.y / tileSize};
+    if (tileSize <= 0) throw new Error("Invalid tile size specified");
```

Comment:
This is not a great UX. If the user types a negative number into the field it crashes the editor. I think you should use drawMissingAsset instead and update the attribute's component so it shows an error.